test/api/bucket/samples.go

```diff
@@ -0,0 +1,20 @@
+package bucket
+
+import (
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+var Buckets = []api.Bucket{
```

Comment:
Just curious, do we need 4 same sample files or it is just a remain from some refactoring?

Reply:
No just few samples. No such significance. Will update